README.md

```diff
@@ -17,26 +17,45 @@ stack. It allows a number of configuration options to customize the logging to y
 
 ### Basic usage
 
-You should be able to use the module by simply adding it as your project's dependency.
+The module comprises two components; simply add a component as your project's dependency to use it.
```

Comment:
there's 3 though

Reply:
There are - unless we remove the (apparently redundant) `dropwizard` component.